pytorch_lightning/loggers/tensorboard.py

```diff
@@ -117,15 +123,12 @@ def log_hyperparams(self, params: Union[Dict[str, Any], Namespace],
             from torch.utils.tensorboard.summary import hparams
             if metrics is None:
                 metrics = {}
-            exp, ssi, sei = hparams(sanitized_params, metrics)
+            exp, ssi, sei = hparams(params, metrics)
             writer = self.experiment._get_file_writer()
```

Comment:
Does this fix #1778? If so, I would add it to the changelog as a separate entry or perhaps make a separate PR if it is not essential to this PR.

Reply:
I mean the update above, I highlighted the wrong part